### PR TITLE
feat(docs): FRMW-168 - Migrate rxCharacterCount docs

### PR DESCRIPTION
**Pull request by bry4n**
JIRA: https://jira.rax.io/browse/FRMW-168

* [x] Dev LGTM
* [x] Dev LGTM

### CODE REVIEW

**Comment by 100stacks** [comment]
LGTM

File: src/rxCharacterCount/README.md [hunk 1]
```diff
@@ -1,39 +1,3 @@
...
-
-But this support should be considered experimental. If you choose to take advantage of it, please ensure you've extensively tested that it performs correctly for your uses.
-
-### Highlighting ###
-Characters that are over the limit will be highlighted in red if the `highlight="true"` attribute is on the directive's element. Because this functionality is currently unstable, it has been left off by default. Please test your use case heavily before shipping with this feature enabled.
-
-Known failure cases:
-* Content that causes a scrollbar in the textarea
-* Initial text (coming from the model) that is over the limit
+A component provides an attribute directive intended for adding to `<textarea>` elements.
```
Comment (CITguy) [inline src/rxCharacterCount/README.md]: Grammar is weird. How about...

```
A component that provides character count functionality and designed for `<textarea>` elements.
```

Comment (bry4n) [inline src/rxCharacterCount/README.md]: updated.

File: src/rxCharacterCount/rxCharacterCount.js [hunk 2]
```diff
@@ -4,7 +4,66 @@
  * @description
  * # rxCharacterCount Component
  *
- * [TBD]
```
Comment (CITguy) [inline src/rxCharacterCount/rxCharacterCount.js]: The only line that should be in this part of the ngdocs should match what is in the readme.

```
A component designed for `<textarea>` elements that provides character count functionality.
```

All other documentation should be combined with the `rxCharacterCount` directive documentation (beginning with ```Place the `rx-character-count` attribute...```)

File: src/rxCharacterCount/rxCharacterCount.js [hunk 3]
```diff
@@ -4,7 +4,66 @@
...
+ * elements. Place the `rx-character-count` attribute into your `<textarea>`, and
+ * a new `<div>` will be added directly underneath it. This directive requires
+ * that you're using `ng-model` with your `<textarea>`
+ *
+ * This `<div>` will watch the content of the `<textarea>`, and display how many
+ * characters are remaining. By default, 254 characters are "allowed". If there
+ * are less than 10 characters remaining, the counter will go orange. If the user
+ * enters more than 254 characters, the counter will go red.
+ *
+ * ### Optional Parameters ###
```
Comment (CITguy) [inline src/rxCharacterCount/rxCharacterCount.js]: This section is unnecessary given that these parameters are already defined and documented in the `rxCharacterCount` directive docs.

File: src/rxCharacterCount/rxCharacterCount.js (hunk 2 above)
Comment (bry4n) [inline src/rxCharacterCount/rxCharacterCount.js]: updated

**Comment by CITguy** [comment]
LGTM

**Pull request merged**